util/ingestor.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+# ================================================================
+# A simple driver for ingestion of anndata to a TileDB group.
+#
+# * Invoke this with no arguments to process:
+#   o canonical input:  ./anndata/pbmc3k_processed.h5ad
+#   o canonical output: ./tiledb-data/pbmc3k_processed/
+#
+# * Invoke this with one argument /path/to/some/somename.h5ad:
+#   o Output will be ./tiledb-data/somename
+#
+# * Invoke this with two arguments to specify input anndata HD5 file
+#   and output TileDB group.
+#
+# Nominal immediate-term support is to local disk, although output to tiledb:/...
+# URIs will be supported.
+#
+# Note this removes and recreates the destination TileDB group on each invocation.
+# ================================================================
+
+from mSCGroup import SCGroup
+import sys, os, shutil
+
+input_path  = 'anndata/pbmc3k_processed.h5ad'
+output_path = 'tiledb-data/pbmc3k_processed'
+if len(sys.argv) == 3:
+    input_path  = sys.argv[1]
+    output_path = sys.argv[2]
+if len(sys.argv) == 2:
+    input_path  = sys.argv[1]
+    # Example 'anndata/pbmc3k_processed.h5ad' -> 'tiledb-data/pbmc3k_processed'
+    output_path = 'tiledb-data/' + os.path.splitext(os.path.basename(input_path))[0]
+
+if not os.path.exists('tiledb-data'):
```

Comment:
Maybe someone else can chime in about this but might want to change to using `tiledb.VFS`'s `is_dir`, `create_dir`, and `remove_dir` in the future so that it can be used on multiple backends

Reply:
ok -- for uploading to `tiledb://...` URIs I suspect we want either:

* Skip the upload, as "already done"
* Use a tiledb-native `overwrite` flag

I used this only for quick iteration on local-disk testing ... I can add a comment to that effect.

Reply:
https://github.com/single-cell-data/TileDB-SingleCell/pull/5/commits/9b426cf421a8d1c3c161ad346a92e6ff21810b34